### DISCUSSION TITLE
confirm deletion of every single waypoint?

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#349] [Q&A] [answered]
Who has requested this? This is terrible. If I have deleted too much waypoints than I can cancel at the end and not press ok. That's fully sufficient. Confirming every single deletion of wps from the waypoint list is just nervy.

**Comment by devemux86** [comment]
Indeed it is better to revert it. :slightly_smiling_face:

**Comment by devemux86** [comment, marked as answer]
Fixed in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
